<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>593</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QScrollArea" name="m_scroll_area">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(128, 128, 128)</string>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>591</width>
         <height>348</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="m_label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="m_menu_bar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>593</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="m_file_menu">
    <property name="title">
     <string>Fichier</string>
    </property>
    <widget class="QMenu" name="m_recent_menu">
     <property name="title">
      <string>Ouvert Récemment</string>
     </property>
     <addaction name="separator"/>
    </widget>
    <addaction name="m_open_action"/>
    <addaction name="m_recent_menu"/>
    <addaction name="separator"/>
    <addaction name="m_quit_action"/>
   </widget>
   <widget class="QMenu" name="menuAller">
    <property name="title">
     <string>Aller à</string>
    </property>
    <addaction name="m_next_image"/>
    <addaction name="m_prev_image"/>
    <addaction name="separator"/>
    <addaction name="m_start_diap_fwd"/>
    <addaction name="m_stop_diap"/>
   </widget>
   <widget class="QMenu" name="menu_diion">
    <property name="title">
     <string>Édition</string>
    </property>
    <addaction name="m_delete_image"/>
   </widget>
   <widget class="QMenu" name="menuAfficher">
    <property name="title">
     <string>Afficher</string>
    </property>
    <widget class="QMenu" name="menuZoom">
     <property name="title">
      <string>Zoom</string>
     </property>
     <addaction name="m_scale_up"/>
     <addaction name="m_scale_down"/>
     <addaction name="m_fit_screen"/>
     <addaction name="m_normal_size"/>
    </widget>
    <addaction name="menuZoom"/>
   </widget>
   <addaction name="m_file_menu"/>
   <addaction name="menu_diion"/>
   <addaction name="menuAller"/>
   <addaction name="menuAfficher"/>
  </widget>
  <action name="m_open_action">
   <property name="text">
    <string>Ouvrir</string>
   </property>
  </action>
  <action name="m_quit_action">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="m_next_image">
   <property name="text">
    <string>Image Suivante</string>
   </property>
   <property name="shortcut">
    <string>Right</string>
   </property>
  </action>
  <action name="m_prev_image">
   <property name="text">
    <string>Image Précédente</string>
   </property>
   <property name="shortcut">
    <string>Left</string>
   </property>
  </action>
  <action name="m_start_diap_fwd">
   <property name="text">
    <string>Démarrer diporama</string>
   </property>
  </action>
  <action name="m_stop_diap">
   <property name="text">
    <string>Arrêter diaporama</string>
   </property>
  </action>
  <action name="m_delete_image">
   <property name="text">
    <string>Supprimer l'image</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="m_scale_up">
   <property name="text">
    <string>Zoom Avant</string>
   </property>
   <property name="shortcut">
    <string>+</string>
   </property>
  </action>
  <action name="m_scale_down">
   <property name="text">
    <string>Zoom Arrière</string>
   </property>
   <property name="shortcut">
    <string>-</string>
   </property>
  </action>
  <action name="m_normal_size">
   <property name="text">
    <string>Taille Normale</string>
   </property>
   <property name="shortcut">
    <string>.</string>
   </property>
  </action>
  <action name="m_fit_screen">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Zoomer au mieux</string>
   </property>
   <property name="shortcut">
    <string>Home</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>m_quit_action</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_open_action</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>openImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_start_diap_fwd</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>startDiap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_stop_diap</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>stopDiap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_prev_image</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>prevImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_scale_down</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>scaleDown()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_scale_up</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>scaleUp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_normal_size</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>normalSize()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_next_image</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>nextImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_fit_screen</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>fitScreen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_delete_image</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>deleteImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>openImage()</slot>
  <slot>nextImage()</slot>
  <slot>prevImage()</slot>
  <slot>startDiap()</slot>
  <slot>stopDiap()</slot>
  <slot>scaleUp()</slot>
  <slot>scaleDown()</slot>
  <slot>normalSize()</slot>
  <slot>fitScreen()</slot>
  <slot>openRecentFile()</slot>
  <slot>deleteImage()</slot>
 </slots>
</ui>
